data/eos-metrics-event-recorder.service.in

```diff
@@ -1,6 +1,5 @@
 [Unit]
 Description=EndlessOS Metrics Event Recorder Server
-Wants=systemd-logind.service
```

Comment:
@cosimoc this is a follow-up to our discussion during the standup in case Jasper and I missed something and/or you're curious what conclusion we came to.